packages/client/src/app/pages/portfolio/port-blog/port-blog.component.html

```diff
@@ -17,7 +17,7 @@ <h2>Blogs</h2>
                     </span>
                 </div>
                 <div class="blog-header">
-                    <h1 class="header"><a [routerLink]="['/portfolio', portUserId, portUserName, 'blog', blog._id]">{{ blog.title }}</a></h1>
+                    <h1 class="header"><a [routerLink]="['/portfolio', portUserId, portUser.username | slugify, 'blog', blog._id]">{{ blog.title }}</a></h1>
```

Comment:
In my testing, I found that portUser.username returns a result that has capitals and spacing, which is different from how it's used in the URL for the user profile and such.

Reply:
slugify normalizes those strings